Sources/LiteSupport/TestRunner.swift

```diff
@@ -53,12 +53,16 @@ class TestRunner {
   /// How to parallelize work.
   let parallelismLevel: ParallelismLevel
 
+  /// The message to print if all the tests passed.
+  let successMessage: String
+
   /// Creates a test runner that will execute all tests in the provided
   /// directory.
   /// - throws: A LiteError if the test directory is invalid.
   init(testDirPath: String?, substitutions: [(String, String)],
        pathExtensions: Set<String>, testLinePrefix: String,
-       parallelismLevel: ParallelismLevel) throws {
+       parallelismLevel: ParallelismLevel,
+       successMessage: String = "All tests passed! 🎉") throws {
```

Comment:
This is nice! 
I think the default here can be removed, since it’s passed in from Run.swift line 39

Reply:
Good idea. I've removed the default in `TestRunner`.